dns/bind/src/opnsense/service/templates/OPNsense/Bind/named.conf

```diff
@@ -186,10 +186,17 @@ logging {
                 print-time yes;
         };
 
-        category default { default_log; };
-        category general { default_log; };
-        category queries { query_log; };
-        category rpz { rpz_log; };
+{% if helpers.exists('OPNsense.bind.general.syslog') %}
```

Comment:
Maybe this should be extended to test not empty

Reply:
Simple `and OPNsense.bind.general.syslog != ""` I assume?